webapp/views/App/views/Data/Charts/state/chartTypes/pie/index.js

```diff
@@ -84,30 +67,28 @@ const pie = {
       },
       other: {
         title: 'Custom Chart',
-        subtitle: 'Custom configuration of the chart',
+        subtitle: 'Configuration of the Chart',
         type: 'container',
         blocks: {
-          'show-title': ShowTitleBlock(),
-          title: TitleBlock(),
-          'show-legend': ShowLegendBlock({
+          title: TitleBlock({
             valuesToSpec: ({ value = [], spec = {} }) => {
-              let legend = {
-                titleFontSize: 8,
-                labelFontSize: 5,
-              }
-
-              if (!value) {
-                legend = false
+              const newSpec = {
+                ...spec,
+                chart: {
+                  ...spec.chart,
+                  title: value,
+                },
               }
-
+              return newSpec
```

Comment:
this object is quite simple, you can just return it, without assigning it into a constant